src/app/item-popup/ItemStats.tsx

```diff
@@ -55,6 +55,19 @@ export default function ItemStats({
   );
 }
 
+// returns the socket associated with a mod that increases a displayed stat, specficially:
+// backup mag (magazine size), counterbalance stock (recoil direction), and targeting adjustor (aim assist)
+function modSocketFor(item) {
+  return (
+    item.sockets &&
+    item.sockets.sockets.find((socket) => {
+      return (
+        socket.plug && [3336648220, 1588595445, 3228611386].includes(socket.plug.plugItem.hash)
```

Comment:
Where possible, we try not to hardcode lists of things. What you can do here is actually look through the mods and find non-masterwork mods that modify stats (see `DimPlug#stats`) and use that. Then when Shadowkeep comes in and changes all the mods, this code will still work!

Reply:
Gotcha, doesnt seem like the `isMasterwork` attribute is working properly? (assuming this is what you meant)
<img width="460" alt="Screen Shot 2019-09-15 at 8 01 39 PM" src="https://user-images.githubusercontent.com/11082871/64929515-0efe0a80-d7f5-11e9-9114-1107359d5a42.png">
For now I've just gone with checking if its a weapon mod
```js
socket.plug.plugItem.itemTypeDisplayName === "Weapon Mod"
```

Reply:
I would actually recommend using `socket.plug.plugItem..itemCategoryHashes.includes(2685412949 )` so that it works with i18n

Reply:
good call 👍